dl/tensorflow/tensorflow-pip.jinja

```diff
@@ -14,4 +14,9 @@ RUN pip --no-cache-dir install --upgrade ${TF_BINARY_URL}
 
 # Install Keras
 RUN pip --no-cache-dir install git+git://github.com/fchollet/keras.git@${KERAS_VERSION}
+
+# Add Tensorboard
+RUN apt-get update && apt-get install -y supervisor
```

Comment:
minor issue, it's better to run a clean up after apt-get install/update to reduce image size, something like: https://github.com/floydhub/dockerfiles/blob/master/base/dl-deps/Dockerfile-py2#L53.